aladdin.sh

```diff
@@ -263,28 +256,26 @@ function handle_ostypes() {
     esac
 }
 
-function prepare_docker_subcommands() {
+function enter_docker_container() {
+
     # if this is not production or staging, we are mounting kubernetes folder so that
     # config maps and other settings can be customized by developers
+    local mounts=""
     if "$DEV"; then
-        DEV_CMD="-v $(pathnorm "$ALADDIN_DIR"):/root/aladdin"
-        DEV_CMD="$DEV_CMD -v /:/minikube_root" # mount the whole minikube system
-        DEV_CMD="$DEV_CMD --workdir /minikube_root$(pathnorm "$PWD")"
+        mounts="$mounts -v $(pathnorm "$ALADDIN_DIR"):/root/aladdin"
```

Comment:
could be good to use an array for this; avoids escaping issues with paths that have spaces in them.  Minor nit though, and would accept it as it is!

Reply:
My bash-fu is weak, would you mind pointing me to an example?

Reply:
SO

The syntax for this would be, e.g.,
```
local -a mounts  # declares $mounts as an array
if "$DEV"; then
    # [@] and [*] are both ways of getting all elements of an array.  Which you use depends on which behaviour you 
    # want when you surround them with quotes --- whether you want the quotes to go around each array element, or 
    # around the whole array.
    # "${mounts[@]}" expands to every element of the array $mounts as a separately quoted string
    #                (ie, (a b "c d") becomes "a" "b" "c d")
    # "${mounts[*])" would be every element concatenated together with spaces
    #                (ie, (a b "c d") becomes "a b c d")
    # Without the surrounding quotes, either version would be treated like a string with spaces in it (ie, broken up into
    # words).  So, for both ${mounts[@]} and ${mounts[*]}:
    #                ((a b "c d") becomes a b c d)
    # [@] is what you want here, as each array element is an argument.

    mounts=("${mounts[@]}" -v $(pathnorm "$ALADDIN_DIR"):/root/aladdin)
fi
....
docker run $FLAGS
....
"${mounts[@]}"
```